src/avatar/avatar.component.ts

```diff
@@ -16,6 +16,12 @@ export const thyAvatarSizeMap = {
     lg: 48
 };
 
+/** https://html.spec.whatwg.org/multipage/embedded-content.html#attr-img-loading */
+type ThyAvatarLoading = 'eager' | 'lazy';
+
+/** https://wicg.github.io/priority-hints/#idl-index */
+type ThyAvatarFetchPriority = 'high' | 'low' | 'auto';
```

Comment:
It's best to export types `ThyAvatarLoading` and `ThyAvatarFetchPriority`

Reply:
Done